src/CentralApi.Contracts/Responses/RelationToLocationRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Middleware.CentralApi.Contracts.Responses;
+
+public class RelationToLocationRequest
+{
+    public Guid robotId { get; set; }
+
+    public List<LocationNames> Locations { get; set; }
```

Comment:
please add `= new List<LocationNames>();` in the end of the line to eliminate the error